server/src/test/java/org/apache/druid/segment/realtime/firehose/InlineFirehoseTest.java

```diff
@@ -147,6 +152,7 @@ public void testCommit()
     Assert.assertSame(Runnables.getNoopRunnable(), result);
   }
 
+  @SuppressWarnings("squid:S00108") // Nested blocks of code should not be left empty
```

Comment:
This is a Sonar thing, and we don't currently use Sonar on a whole-community level, so I'd leave this out for now. But please feel free to raise the suggestion of using Sonar more broadly on the dev list, dev@druid.apache.org. It looks like there is an ASF instance at: https://builds.apache.org/analysis/ that we could use.